contributors/design-proposals/multi-platform.md

```diff
@@ -175,7 +175,7 @@ If someone tries to add a dependency that doesn't satisfy these requirements; th
 
 ### kubectl should be released for all platforms that are relevant
 
-kubectl is released for more platforms than the proposed server platforms, if you want to check out an up-to-date list of them, [see here](../../hack/lib/golang.sh).
+kubectl is released for more platforms than the proposed server platforms, if you want to check out an up-to-date list of them, [see here](https://github.com/kubernetes/kubernetes/blob/master/hack/lib/golang.sh).
```

Comment:
Please update this link to `https://git.k8s.io/kubernetes/hack/lib/golang.sh`. Thanks!

Reply:
thanks for review.
fix it